pkg/logs/logs.go

```diff
@@ -176,3 +172,15 @@ func WithInfof(l logr.Logger) *LogWithFormat {
 func (l *LogWithFormat) Infof(format string, a ...interface{}) {
 	l.Info(fmt.Sprintf(format, a...))
 }
+
+func initDeprecatedFlags(fs *flag.FlagSet) {
+	var allFlags flag.FlagSet
+	klog.InitFlags(&allFlags)
+	allFlags.VisitAll(func(f *flag.Flag) {
+		switch f.Name {
+		case "add_dir_header", "alsologtostderr", "log_backtrace_at", "log_dir", "log_file", "log_file_max_size",
+			"logtostderr", "one_output", "skip_headers", "skip_log_headers", "stderrthreshold":
+			fs.Var(f.Value, f.Name, f.Usage)
+		}
+	})
+}
```

Comment:
@maelvls I've added this function to init general klog flags. This might be easier to cleanup later. What do you think?

Reply:
That works well. Thanks!